posts/deployToEcsWithGithubAction.md

```diff
@@ -0,0 +1,177 @@
+---
+authors:
+- Amine Ouali Alami
+tags:
+- Spring Boot
+- JHipster
+- GitHub
+- GitHub Actions
+- DevOps
+- AWS
+- Amazon ECS
+- AWS Fargate
+- Amazon ECR
+
+date: 2020-20-10T20:16:10.000Z
+title: "Deploy to ECS using GitHub Actions"
+image: https://raw.githubusercontent.com/ippontech/blog-usa/master/images/2020/01/deployToEcsWithGithubAction.jpg
+---
+
+Building a DevOps platform nowadays can become a challenge for architecture and platform infrastructure teams due to the integration complexity with many platforms: Source Control, Continuous Integration/Continuous Delivery (CI/CD), Artifact Repositories, Quality, SAST/SCA, and Deployment. Many DevOps leaders believe that the challenge of DevOps is to ultimately reduce the number of platforms involved in one single pipeline.
```

Comment:
Please spell out initialisms prior to using them. Also, decapitalize nouns unless they are proper nouns.

Reply:
I changed SAST/SCA by just security.
I don't want here to spell out: Static Application Security Testing (SAST), Dynamic Application Security Testing (DAST), Security Control Assessment (SCA), Seems too much for me.